### PR TITLE
{toolchain} intel/2019.01

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/34b8ab1c83a96687ad10ab956a68ae47 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6dc80dc05107e1ee7f526e92e9242f49 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/33ac40aa10b3ac7e937f5e90e7c94390 for a full test report.

File: easybuild/easyconfigs/i/icc/icc-2019.1.144-GCC-8.2.0-2.31.1.eb [hunk 1]
```diff
@@ -8,6 +8,7 @@ description = "Intel C and C++ compilers"
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14865/']
 sources = ['parallel_studio_xe_%(version_major)s_update%(version_minor)s_composer_edition.tgz']
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/i/icc/icc-2019.1.144-GCC-8.2.0-2.31.1.eb]: `s/edition/edition_for_cpp`

Comment (boegel) [reply easybuild/easyconfigs/i/icc/icc-2019.1.144-GCC-8.2.0-2.31.1.eb]: Ouch... That, and I need to fix the checksum too. On it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/dbc2a2d31ed533bc6991b9aa0c1abd31 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/cfff96eadd3571d553a51232e186afc6 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**